cts/cts-cli.in

```diff
@@ -1226,7 +1226,7 @@ for t in $tests; do
     test_$t > "$TMPFILE"
 
     sed -e 's/cib-last-written.*>/>/'\
-        -e 's/ last-run=\"[0-9]*\"//'\
+        -e 's/ last-run=\"[A-Za-z0-9: ]*\"//'\
```

Comment:
We'll need to comment somewhere why this isn't just numeric -- for the CIB status section, they will always be numeric, it's just crm_mon that makes them human-friendly.

Reply:
Done.